src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -172,8 +172,9 @@ export function PharmacyCard(props: PharmacyProps) {
   };
 
   const badgeMarkup = () => {
-    const returnTags: JSX.Element[] = [];
+    let returnTags: JSX.Element[] = [];
     Object.keys(props.vaccineAvailabilities).forEach((availability) => {
+      returnTags = [];
```

Comment:
`returnTags` should be defined inside this code block, instead of being reset to `[]` on each iteration of the loop. Variables should be scoped to where they are used.